cypress/support/helper.ts

```diff
@@ -53,72 +21,78 @@ export function continuePressingTABKey(count: number, shift = false) {
   }
 }
 
-export function positionOfElement(type: string) {
-  return {
-    first: "0",
-    second: "1",
-    third: "2",
-    fourth: "3",
-    fifth: "4",
-    sixth: "5",
-    seventh: "6",
-    eighth: "7",
-    ninth: "8",
-    tenth: "9",
-    eleventh: "10",
-    thirteenth: "13",
-    nineteenth: "19",
-  }[type];
-}
+const positions = {
+  first: 0,
+  second: 1,
+  third: 2,
+  fourth: 3,
+  fifth: 4,
+  sixth: 5,
+  seventh: 6,
+  eighth: 7,
+  ninth: 8,
+  tenth: 9,
+  eleventh: 10,
+  thirteenth: 12,
+  nineteenth: 18,
+};
 
-export function positionOfPaginationButton(type: string) {
```

Comment:
is this function not used anywhere?

Reply:
Searching with VSCode I didn't find any files with references to this function, which is why I decided to remove it. Looking at the git history, it seems this function was created for use in the old BDD cypress tests, but has since become unused.